subjects/error_types/README.md

```diff
@@ -15,7 +16,7 @@ Examples:
 Create a structure named `Form` that will have the following fields:
 - `first_name`: `String`
 - `last_name`: `String`
-- `birth`: `NaiveDate` that will convert a string `"2015-09-05"` to a date of that format.
+- `birth`: `NaiveDate` (converted from a string using the create_date function).
```

Comment:
The description here have no meaning since we have a data type `NaiveDate`.
Please remove the description from here then i can approve your PR